packages/web/src/pages/collection-page/components/desktop/CollectionPage.tsx

```diff
@@ -250,36 +280,65 @@ const CollectionPage = ({
           <EmptyPage isOwner={isOwner} text={customEmptyText} />
         ) : (
           <div className={styles.tableWrapper}>
-            <TracksTable
-              key={playlistName}
-              loading={tracksLoading}
-              loadingRowsCount={trackCount}
-              columns={columns}
-              userId={userId}
-              playing={playing}
-              playingIndex={playingIndex}
-              dataSource={dataSource}
-              allowReordering={
-                userId !== null &&
-                userId === playlistOwnerId &&
-                allowReordering &&
-                !isAlbum
-              }
-              onClickRow={onClickRow}
-              onClickFavorite={onClickSave}
-              onClickTrackName={onClickTrackName}
-              onClickArtistName={onClickArtistName}
-              onClickRepost={onClickRepostTrack}
-              onSortTracks={onSortTracks}
-              onReorderTracks={onReorderTracks}
-              onClickRemove={isOwner ? onClickRemove : null}
-              removeText={`${messages.remove} ${
-                isAlbum ? messages.type.album : messages.type.playlist
-              }`}
-            />
-            {collectionLoading && typeTitle === 'Audio NFT Playlist' ? (
-              <LoadingSpinner className={styles.spinner} />
-            ) : null}
+            {isNewTablesEnabled ? (
+              <TableComponent
+                // @ts-ignore
```

Comment:
🤔

Reply:
This was due to artistName not being present in both the nft playlist table columns and the tracks table columns. Not sure what the best way to guarantee that the columns correspond to the correct type of table